src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Properties/InterceptedProjectProperties/StartupObjectValueProvider.cs

```diff
@@ -6,8 +6,8 @@
 namespace Microsoft.VisualStudio.ProjectSystem.VS.Properties;
 
 // The AppliesTo metadata has no effect given the limitations described in https://github.com/dotnet/project-system/issues/8170.
-[ExportInterceptingPropertyValueProvider(StartupObjectProperty, ExportInterceptingPropertyValueProviderFile.ProjectFile)]
-[AppliesTo(ProjectCapability.VisualBasic)]
+[ExportInterceptingPropertyValueProvider(InterceptedStartupObjectProperty, ExportInterceptingPropertyValueProviderFile.ProjectFile)]
+[AppliesTo(ProjectCapability.WPF + "&" + ProjectCapability.WindowsForms)]
```

Comment:
This means the interceptor would only apply to projects with _both_ the WPF and WindowsForms capabilities; is that what you want? If you want it to apply to projects that have _either_ WPF or WindowsForms, you need to do this instead:

`[AppliesTo(ProjectCapability.WPF + "|" + ProjectCapability.WindowsForms)]`

Reply:
Oh, my bad, that was what I was thinking. Thanks for catching it.